File: backend/models/BookingRm.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace backend.models
{
    public record BookingRm(
        Guid Id,
        string Airline,
        double Price,
        TimePlaceRm Departure,
        TimePlaceRm Arrival,
        int NumberOfSeats,
        string PassengerEmail);
}

File: frontend/src/app/api/models/booking-rm.ts
import { TimePlaceRm } from './time-place-rm';

export interface bookingRm extends TimePlaceRm {
  id: string;
  airline: string;
  price: number;
  departure: TimePlaceRm;
  arrival: TimePlaceRm;
  numberOfSeats: number;
  passengerEmail: string;
}
